solidity/ecdsa/deploy/06_transfer_ownership_random_beacon_chaosnet.ts

```diff
@@ -0,0 +1,21 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { getNamedAccounts, helpers } = hre
+  const { deployer, governance } = await getNamedAccounts()
+
+  await helpers.ownable.transferOwnership(
+    "RandomBeaconChaosnet",
+    governance,
+    deployer
+  )
+}
+
+export default func
+
+func.tags = ["RandomBeaconChaosnetTransferOwnership"]
+func.dependencies = ["RandomBeaconChaosnet"]
+
+func.skip = async (hre: HardhatRuntimeEnvironment): Promise<boolean> =>
+  !hre.network.tags.useRandomBeaconChaosnet
```

Comment:
We will have to execute this step and the previous one on mainnet during the deployment. Can we add `useRandomBeaconChaosnet` tag for the `mainnet` network in `hardhat.config.ts`?

Reply:
Done in 3f949159d96a4aa599e63931a95c1917fc1e6823.